depends/common/libzvbi/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ else()
   list(APPEND zvbi_conf CFLAGS=-I${CMAKE_INSTALL_PREFIX}/include)
 endif()
 list(APPEND zvbi_conf LDFLAGS=-L${CMAKE_INSTALL_PREFIX}/lib)
-if (CORE_SYSTEM_NAME STREQUAL android)
+if ((CORE_SYSTEM_NAME STREQUAL android) OR (CMAKE_CXX_COMPILER MATCHES arm-webos))
```

Comment:
Do we not have a `CORE_SYSTEM_NAME` for webos?

Reply:
Good question. I think CORE_SYSTEM_NAME might be equal to linux, because that is what it is. This is the workaround we came up with for inputstream.adaptive.

Reply:
Yeh just confirmed "CORE_SYSTEM_NAME is: linux"